spec/amp-email-format.md

```diff
@@ -179,6 +186,26 @@ amphtml-validator --html_format AMP4Email \
 amp_email.html
 ```
 
+## Privacy and Security
+
+AMPHTML Emails track email opens with pixel tracking techniques, same as regular emails. Any user-initiated requests for data from external services will also indicate the user is interacting with the message. Email clients may offer their users the ability to disable loading remote images, and other external requests.
```

Comment:
I'd split into section `### Tracking email opens and interaction`. Also, let's change back to "AMPHTML Emails **can** track opens" since it's not something that's present by design.